File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/java/javimlopez.java

public class javimlopez  {

    public static void main(String[] args){
        fizzbuzz100();
    }

    public static void fizzbuzz100 () {
        for(int i = 1; i<=100; i++){
            if((i%3 == 0) && (i%5 == 0)){
                System.out.println("fizzbuzz");
                continue;
            }
            if(i%3 == 0){
                System.out.println("fizz");
                continue;
            }else if(i%5 == 0){
                System.out.println("buzz");
                continue;
            }

            System.out.println(i);
        }
    }
}

File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/c/javimlopez.c
#include <stdio.h>

void fizzbuzz(int n) {
  int i = 1;
  while (i <= n) {

    if (i % 3 == 0 && i % 5 == 0) {
      printf("fizzbuzz\n");
    } else if (i % 5 == 0) {
      printf("buzz\n");
    } else if (i % 3 == 0) {
      printf("fizz\n");
    } else
      printf("%d\n", i);
    i++;
  }
}

int main() {
  int n = 100;
  fizzbuzz(n);

  return 0;
}
